src/js/contexts/AnnounceContext/announce-context.stories.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { storiesOf } from '@storybook/react';
+import { grommet } from 'grommet/themes';
+import { Box, Grommet, Text, AnnounceContext } from 'grommet';
+
+class Announcer extends Component {
+  static propTypes = {
+    announce: PropTypes.func.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    const { message, role, politenessSetting } = this.props;
+    this.state = {
+      message:
+        message || 'Here is a simple announcement. This will soon disappear',
+      politenessSetting: politenessSetting || 'polite',
```

Comment:
Can we align with the function parameter and call this `mode`?

Just as an aside, all props are "settings" of some sort, so the suffix "Setting" seems extraneous.

Reply:
Sure, `mode` is perfect. Thanks!!

Reply:
Addressed in recent commit. Thanks !!